guides/cloud/aws_sandbox.md

```diff
@@ -4,64 +4,64 @@
 
 The Made Tech AWS Sandbox accounts are primarily used for individuals' personal development and testing. We run [AWS SSO](https://aws.amazon.com/single-sign-on/) to simplify and secure the IAM aspect of accessing this account.
 
-Currently there are 2 accounts you can use:
+There are 2 accounts you can use:
 - [Playground account](#playground-account)
 - [Devops Pairing Interviews Account](#devops-pairing-interviews-account)
 
-# Playground account
+## Playground account
```

Comment:
Fixed the hierarchy here whilst trying to figure out what the link was being reported as dead.